src/content/docs/get-started/index.mdx

```diff
@@ -83,9 +83,9 @@ The `mountpoint` is the URL that points to your content folder on Google Drive o
 
 ## Example
 
-Our WKND demo site was built from the same boilerplate you will set up to develop your own storefront. You can access the full WKND demo site here: https://main--xsc-wknd-commerce--hlxsites.hlx.live/.
+Our CitiSignal demo site was built from the same boilerplate you will set up to develop your own storefront. You can access the full demo site here: https://main--citisignal--adobedevxsc.hlx.page/.
 
-<Embed src="https://main--xsc-wknd-commerce--hlxsites.hlx.live/" height="520" />
+<Embed src="https://main--citisignal--adobedevxsc.hlx.page/" height="520" />
```

Comment:
Did you mean to use `.page`? Also I think we can and should start using `aem.live` or `aem.page`.

cc @herzog31

Reply:
I actually don't know the difference between the two 😅. I just used the link that Olena sent me. 

Is this the preferred link?

https://main--citisignal--adobedevxsc.aem.live/

Reply:
IIUC there are caching differences, but also there can be differences such as which backend is used.

We map .live to production environment, .page to staging, and localhost to integration cloud environments, so depending on which url you use, you may get different reuslts. I would double check with Olena or even the sales demo team working on this, which URL they would like to have. I added you to the internal slack channel.

Reply:
Confirmed with Olena that it should be `https://main--citisignal--adobedevxsc.aem.live/`.